manager/manager.go

```diff
@@ -724,10 +724,13 @@ func (m *Manager) watchForClusterChanges(ctx context.Context) error {
 func (m *Manager) rotateRootCAKEK(ctx context.Context, clusterID string) error {
 	// If we don't have a KEK, we won't ever be rotating anything
 	strPassphrase := os.Getenv(ca.PassphraseENVVar)
-	if strPassphrase == "" {
+	strPassphrasePrev := os.Getenv(ca.PassphraseENVVarPrev)
+	if strPassphrase == "" && strPassphrasePrev == "" {
 		return nil
 	}
-	strPassphrasePrev := os.Getenv(ca.PassphraseENVVarPrev)
+	log.G(ctx).Warn("Encrypting the root CA key in swarm using environment variables is deprecated. " +
+		"Support for decrypting or rotating the key will be removed in the future.")
```

Comment:
I feel like this should only be logged if `strPassphrase` is non-empty. If we're only decrypting, we'll commit the decrypted version, and the user doesn't have to take any further action.